cmux-tui/spec/commands.md

```
@@ -3528,7 +3528,7 @@ CLI mapping:
 | Item | Value |
 | --- | --- |
 | Verb | `list-agents` |
-| Flags | `[--surface <id>] [--state working|blocked|idle|done|unknown]` |
+| Flags | `[--surface <id>] [--state working|blocked|idle|done|interrupted|unknown]` |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)